src/cmd/migrate.ts

```diff
@@ -372,16 +372,46 @@ export const applyChanges = async (
 
 export const unparsePaths = (path: string, values: Record<string, any>): Array<string> => {
   if (path.includes('{team}')) {
-    const paths: Array<string> = []
+    let paths: Array<string> = []
     const teams: Array<string> = Object.keys(values?.teamConfig as Record<string, any>)
     teams.forEach((teamName) => paths.push(path.replace('{team}', teamName)))
+    paths = isArray(paths, values)
     return paths.sort()
   } else {
-    return [path]
+    const paths = isArray([path], values)
+    return paths
   }
 }
+
+function isArray(paths: string[], values: Record<string, any>): string[] {
```

Comment:
It is not clear to me what exactly this function does, but I would expect anything that starts with `is` to be a function with a boolean return type. Also there is a built-in function `Array.isArray` which I would advise not to reuse.